### PR TITLE
Update tools from openstack-charm-testing

### PR DESCRIPTION
**Pull request by fnordahl**
Tests will pass as soon as this is landed: openstack-charmers/openstack-charm-testing#6

### CODE REVIEW

**Comment by andrewdmcleod** [comment]
testing this, will update

File: development/openstack-telemetry-xenial-queens/openrc [hunk 1]
```diff
@@ -1 +1 @@
-../shared/openrc
\ No newline at end of file
+openrcv3_project
```
Comment (ajkavanagh) [inline development/openstack-telemetry-xenial-queens/openrc]: this looks different? should it be a link?

File: development/openstack-telemetry-bionic-queens/openrc [hunk 2]
```diff
@@ -1 +1 @@
-../shared/openrc
\ No newline at end of file
+openrcv3_project
```
Comment (ajkavanagh) [inline development/openstack-telemetry-bionic-queens/openrc]: This looks different too?  should it be a link?

File: development/openstack-lxd-xenial-queens/openrc [hunk 3]
```diff
@@ -1 +1 @@
-../shared/openrc
\ No newline at end of file
+openrcv3_project
```
Comment (ajkavanagh) [inline development/openstack-lxd-xenial-queens/openrc]: Not shared link?

File: development/openstack-base-xenial-queens/openrc [hunk 4]
```diff
@@ -1 +1 @@
-../shared/openrc
\ No newline at end of file
+openrcv3_project
```
Comment (ajkavanagh) [inline development/openstack-base-xenial-queens/openrc]: not shared link?

File: development/openstack-base-bionic-queens/openrc [hunk 5]
```diff
@@ -1 +1 @@
-../shared/openrc
\ No newline at end of file
+openrcv3_project
```
Comment (ajkavanagh) [inline development/openstack-base-bionic-queens/openrc]: Not shared link?

File: stable/openstack-base/openrc [hunk 6]
```diff
@@ -1 +1 @@
-../shared/openrc
\ No newline at end of file
+openrcv3_project
```
Comment (ajkavanagh) [inline stable/openstack-base/openrc]: Not a link?

File: stable/openstack-lxd/openrc [hunk 7]
```diff
@@ -1 +1 @@
-../shared/openrc
\ No newline at end of file
+openrcv3_project
```
Comment (ajkavanagh) [inline stable/openstack-lxd/openrc]: Not a link?

File: stable/openstack-telemetry/openrc [hunk 8]
```diff
@@ -1 +1 @@
-../shared/openrc
\ No newline at end of file
+openrcv3_project
```
Comment (ajkavanagh) [inline stable/openstack-telemetry/openrc]: Not a link?

File: development/openstack-telemetry-bionic-queens/openrc (hunk 2 above)
Comment (andrewdmcleod) [reply development/openstack-telemetry-bionic-queens/openrc]: its fine, it is a link - to a file in the same directory

File: development/openstack-telemetry-xenial-queens/openrc (hunk 1 above)
Comment (andrewdmcleod) [reply development/openstack-telemetry-xenial-queens/openrc]: its fine, it is a link - to a file in the same directory

**Comment by ajkavanagh** [comment]
Turns out I was mistaken; good to go.

**Pull request merged**